src/ActiveSpeakerObserver.ts

```diff
@@ -0,0 +1,66 @@
+import { Logger } from './Logger';
+import { EnhancedEventEmitter } from './EnhancedEventEmitter';
+import { RtpObserver } from './RtpObserver';
+import { Producer } from './Producer';
+
+export interface ActiveSpeakerObserverOptions {
+
+	interval?: number;
+	/**
+	 * Custom application data.
+	 */
+	appData?: any;
+}
+
+export interface ActiveSpeakerObserverActivity {
+	/**
+	 * The producer instance
+	 */
+	producer: Producer;
+}
+
+const logger = new Logger('ActiveSpeakerObserver');
+
+export class ActiveSpeakerObserver extends RtpObserver
+{
+	/**
+	 * @private
+	 */
+	constructor(params: any)
+	{
+		super(params);
+		this._handleWorkerNotifications();
+	}
+
+	/**
+	 * Observer.
+	 */
+	get observer(): EnhancedEventEmitter
+	{
+		return this._observer;
+	}
+
+	private _handleWorkerNotifications(): void
+	{
+		this._channel.on(this._internal.rtpObserverId, (event: string, data?: any) =>
+		{
+			switch (event)
+			{
+				case 'dominantSpeaker':
+				{
+					const dominantSpeaker = {
+						producer : this._getProducerById(data.producerId)
+					};
+
+					this.safeEmit('dominantSpeaker', dominantSpeaker);
```

Comment:
Please use lowcase in JS event names as convention.

```suggestion
					this.safeEmit('dominantspeaker', dominantSpeaker);
```

NOTE: This should be changed in tests as well obviously.